tcp-native/src/main/java/org/ballerinalang/stdlib/tcp/SslHandshakeEventHandler.java

```diff
@@ -0,0 +1,51 @@
+package org.ballerinalang.stdlib.tcp;
+
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.handler.flow.FlowControlHandler;
+import io.netty.handler.ssl.SslHandshakeCompletionEvent;
+import org.ballerinalang.stdlib.tcp.nativeclient.Client;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Class to handler ssl handshake event.
+ */
+public class SslHandshakeEventHandler extends ChannelInboundHandlerAdapter {
+    private TcpListenerHandler tcpListenerHandler;
+    private TcpClientHandler tcpClientHandler;
+    private static final Logger log = LoggerFactory.getLogger(Client.class);
+
+    public SslHandshakeEventHandler(TcpListenerHandler handler) {
+        tcpListenerHandler = handler;
+    }
+
+    public SslHandshakeEventHandler(TcpClientHandler handler) {
+        tcpClientHandler = handler;
+    }
+
+
+    @Override
+    public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
```

Comment:
Shall we use meaningful variable names like `event` instead of names like `evt`?

Reply:
+1